src/worker/startup.ts

```diff
@@ -93,6 +94,11 @@ const setDelayOnRateLimiting = (jobHandler) => async (job: Job, logger: LoggerWi
 };
 
 const sendQueueMetrics = async () => {
+	// Only send queue metrics in prod
+	if (!isNodeProd()) {
+		return;
+	}
+
```

Comment:
I kept getting "sending queue metrics" logs in my dev, so we now only send them in production mode.